### PR TITLE
Create JOHNY.py

### PR DESCRIPTION
**Pull request by chandbud5**
**Fixes issue:**
#4668 

**Changes:**
UNCLE JOHNY problem in python

### CODE REVIEW

File: code/online_challenges/src/codechef/JOHNY/JOHNY.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+t = int(input())
+for x in range(t):
+    n = int(input())
+    p = list(map(int, input().split()))
+    k = int(input())
+    a = p[k-1]
```
Comment (arnavb) [inline code/online_challenges/src/codechef/JOHNY/JOHNY.py]:
```suggestion
    a = p[k - 1]
```

File: code/online_challenges/src/codechef/JOHNY/JOHNY.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+t = int(input())
+for x in range(t):
+    n = int(input())
+    p = list(map(int, input().split()))
+    k = int(input())
+    a = p[k - 1]
+    p.sort()
+    print(p.index(a)+1)
```
Comment (arnavb) [inline code/online_challenges/src/codechef/JOHNY/JOHNY.py]:
```suggestion
    print(p.index(a) + 1)
```

**Comment by chandbud5** [comment]
Will this contribution be counted for my GSSoC 20??

**Comment by arnavb** [comment]
@chandbud5 I'm not sure how that works. I only maintain this repository; you'll have to ask @AdiChat.

**Pull request merged**